cpg-concepts/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Extension.kt

```diff
@@ -47,3 +49,11 @@ val Node.conceptNodes: Set<Concept>
  */
 val Node.operationNodes: Set<Operation>
     get() = this.nodes.flatMapTo(mutableSetOf()) { it.overlays.filterIsInstance<Operation>() }
+
+/** Returns a [Set] of all subclasses of [Concept]. */
+fun listConceptClasses(): Set<Class<out Concept>> =
+    Reflections("de.fraunhofer.aisec", SubTypes).getSubTypesOf(Concept::class.java)
+
+/** Returns a [Set] of all subclasses of [Concept]. */
+fun listOperationClasses(): Set<Class<out Operation>> =
```

Comment:
Wouldnt' it make more sense to turn this into a refined generic function, so I can potentially call this on all (sub)-classes of operations/concepts,overlays,etc.

Reply:
I'll change this.

I'm also not sure how we can improve this filter for the package. Get rid of it completely or make it configurable by an argument?

Reply:
Maybe as parameter with the current value as default. Also listOverlays is misleading. OverlayClasses?